Sources/JWT/Signers/RSASigner.swift

```diff
@@ -26,67 +26,117 @@ extension HashMethod {
     }
 }
 
-public struct RS256: RSASigner {
-    public let key: Bytes
+public typealias CRSAKey = UnsafeMutablePointer<RSA>
```

Comment:
Hmm.... since `RSAKey` is in the `RSASigner` protocol I can't make it private. :\ Any ideas?

```swift
public protocol RSASigner: Signer, BytesInitializable {
    var key: RSAKey { get }
    var hashMethod: HashMethod { get }
    init(key: Bytes) throws
}
```